### PR TITLE
Feature: speed up transaction

### LINKED ISSUE #1107: Speed up transactions
We should add the same feature that we have in the extension to be able to speed up txs that have been submitted and taking longer than x

### PR DESCRIPTION
**Pull request by estebanmino**
**Description**

Adds ability to speed up transactions

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #1107

### CODE REVIEW

File: locales/es.json [hunk 1]
```diff
@@ -480,7 +481,14 @@
...
 		"no_address_for_ens": "No existe dirección para nombre ENS",
-		"approve_warning": "Al aprobar esta acción, otorgas permiso para que este contrato gaste hasta"
+		"approve_warning": "Al aprobar esta acción, otorgas permiso para que este contrato gaste hasta",
+		"cancel_tx_title": "¿Intentar cancelar?",
+		"cancel_tx_message": "Enviar este intento no garantiza que su transacción original será cancelada. Si el intento de cancelación es exitoso, se le cobrará la tarifa de transacción anterior.",
+		"speed_up_tx_title": "¿Intentar acelerar?",
+		"speed_up_tx_message": "nviar este intento no garantiza que su transacción original se acelerará. Si el intento de acelerar es exitoso, se le cobrará la tarifa de transacción anterior.",
+		"nevermind": "No importa",
+		"gas_cancel_fee": "Tarifa de Cancelación",
+		"gas_speed_up_fee": "Tarifa de Aceleración "
```
Comment (brunobar79) [inline locales/es.json]: Extra space at the end

**Review by brunobar79 — APPROVED** [review]
This is great!  💯  🔥  🙌

**Comment by ibrahimtaveras00** [comment]
**Issue 1:**

Speed up notification 
image

**Issue 2:**

Went to kyberswap.com to transfer some DAI and when I attempted to speed up the txn, tapping on the CTA didn't do anything and there was a warning; seen here = https://recordit.co/yJkggYty9x

Screen Shot 2019-11-05 at 3 58 42 PM

**Issue 3:**

Went to uniswap trying to get some DAI, and I went to edit my gas to be slow, upon going back to the confirm screen and tapping on the confirm button, I get this; seen here = https://recordit.co/qoY7XwGEnK

image

File: locales/es.json [hunk 2]
```diff
@@ -480,7 +481,14 @@
 		"rate_not_available": "Conversión no disponible",
 		"optional": "Opcional",
 		"no_address_for_ens": "No existe dirección para nombre ENS",
-		"approve_warning": "Al aprobar esta acción, otorgas permiso para que este contrato gaste hasta"
+		"approve_warning": "Al aprobar esta acción, otorgas permiso para que este contrato gaste hasta",
+		"cancel_tx_title": "¿Intentar cancelar?",
+		"cancel_tx_message": "Enviar este intento no garantiza que su transacción original será cancelada. Si el intento de cancelación es exitoso, se le cobrará la tarifa de transacción anterior.",
+		"speed_up_tx_title": "¿Intentar acelerar?",
+		"speed_up_tx_message": "nviar este intento no garantiza que su transacción original se acelerará. Si el intento de acelerar es exitoso, se le cobrará la tarifa de transacción anterior.",
+		"nevermind": "No importa",
```
[the remark this answers is not in the document]
Comment (estebanmino) [reply locales/es.json]: 🤷‍♂

**Review by ibrahimtaveras00 — APPROVED** [review]
All fixes look good, QA Passed 👍

**Pull request merged**